src/configure

```diff
@@ -27,7 +27,7 @@ CONFIGURE_VERSION=2
 
 function rel2abs {
   if [ ! -z "$1" ]; then
-    local retval=`cd $1 2>/dev/null && pwd || exit 1`
```

Comment:
OK, then I guess the correct solution would be either test existence of the directory (and return 1, if it does not exist)